src/main/java/org/sagebionetworks/bridge/models/activities/ActivityEventObjectType.java

```diff
@@ -149,6 +149,27 @@ public String getEventId(String objectId, ActivityEventType eventType, String an
         public String getEventId(String objectId, ActivityEventType eventType, String answerValue) {
             return this.name().toLowerCase();
         }
+    },
+    
+    /**
+     * An event created the first time the participant signs in to the Bridge server via 
+     * any authentication method.
+     */
+    FIRST_SIGN_IN(IMMUTABLE) {
+        public String getEventId(String objectId, ActivityEventType eventType, String answerValue) {
+            return this.name().toLowerCase();
+        }
+    },
```

Comment:
Rename first_app_sign_in, OR email everyone and verify that someone wants this. Because as Nate notes, it could be confusing for schedulers in a multi-study scenario.

Reply:
removed